pages/_app.js

```diff
@@ -1,15 +1,26 @@
 import GlobalStyle from '../components/GlobalStyle';
 import Layout from '../components/Layout';
-import { useState } from 'react';
+// import { useState } from 'react';
+import useLocalStorage from '../hooks/useLocalStorage';
 import collection from '../json/data-dummy.json';
 
 function MyApp({ Component, pageProps }) {
-  const [collectionState, setCollectionState] = useState(() =>
-    collection.map((file) => {
-      return { ...file, isChecked: false };
-    })
+  // const [collectionState, setCollectionState] = useState(() =>
+  //   collection.map((file) => {
+  //     return { ...file, isChecked: false };
+  //   })
+  // );
```

Comment:
you can remove parts where there are only comments

Reply:
Ah right, yes removed it. Thank you!